tooling/Microsoft.VisualStudio.RazorExtension/source.extension.vsixmanifest

```diff
@@ -26,6 +26,7 @@
     <Asset Type="Microsoft.ServiceHub.Service" d:Source="File" Path="razorLanguageService64.servicehub.service.json" />
     <Asset Type="Microsoft.VisualStudio.Assembly" Path="Microsoft.AspNetCore.Mvc.Razor.Extensions.dll" />
     <Asset Type="Microsoft.VisualStudio.Assembly" Path="Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X.dll" />
+    <Asset Type="Microsoft.VisualStudio.Assembly" Path="Microsoft.AspNetCore.Mvc.Razor.Extensions.Version2_X.dll" />
```

Comment:
How do I test this in VS - install the VSIX and try a 2.1 project?

Reply:
you can just F5 the VSIX project and then try and 3.0 and 2.1 project